evidence.go

```diff
@@ -200,15 +202,28 @@ func (e *CcaEvidence) Verify(iak crypto.PublicKey, rpk crypto.PublicKey) error {
 		return fmt.Errorf("missing CCA realm Token")
 	}
 
+	//	var rpk crypto.PublicKey
```

Comment:
Please remove commented code!

Reply:
good catch 👍 

done in cdb2e74673f90dc2f6213eb9141f455000454ac7